src/jit/gentree.cpp

```diff
@@ -1740,6 +1740,35 @@ bool GenTreeCall::IsHelperCall(Compiler* compiler, unsigned helper) const
     return IsHelperCall(compiler->eeFindHelper(helper));
 }
 
+void GenTreeCall::ReplaceCallOperand(GenTree** operandUseEdge, GenTree* replacement)
```

Comment:
> This method needs a header comment.

Will do.

> It appears that it want to handle every modification of call "operands" including the gtControlExpr, gtCallCookie and gtCallAddr, although these last three don't actually require this method to do any extra work.

That's correct: the caller may not know whether or not the operand that it is replacing is an argument (and therefore needs an arg table update). It is this method's responsibility to decide.